From a83ec0c045d3fadb78a879d0ac8ae999b89786df Mon Sep 17 00:00:00 2001
From: fuzzard <bmurphy@bcmcs.net>
Date: Sun, 24 Feb 2019 07:47:31 +1000
Subject: [PATCH] alignas c++ fix

alignas is a keyword in c++. Dont define.
Change is in windows header already, and GCC 8.3.0 is erroring on this now.
---
 hdhomerun_os_posix.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hdhomerun_os_posix.h b/hdhomerun_os_posix.h
index baab8e5..d3dc5cb 100644
--- a/hdhomerun_os_posix.h
+++ b/hdhomerun_os_posix.h
@@ -53,7 +53,7 @@ typedef struct {
 
 #define LIBHDHOMERUN_API
 
-#if !defined(alignas)
+#if !defined(alignas) && !defined(__cplusplus)
 #define alignas(n) __attribute__((aligned(n)))
 #endif
 
